cmstestsuite/TestCleanCheckout.py

```diff
@@ -77,7 +77,8 @@ def setup_cms():
     os.environ["PYTHONPATH"] = "%(TEST_DIR)s" % CONFIG
 
     info("Building cms.")
-    sh("./setup.py build")
+    sh("./prerequisites.py compile_l10n")
+    sh("./prerequisites.py compile_isolate")
```

Comment:
Are you suggesting to keep both "setup build" and "prerequisite compile"? I removed "setup build" because it didn't seem to be necessary. With setuptools (at least on version 15 and up) the `build` command seems to be unnecessary.